gpiozero/spi_devices.py

```diff
@@ -128,10 +167,9 @@ def channel(self):
     @property
     def differential(self):
         """
-        If ``True``, the device is operated in pseudo-differential mode. In
-        this mode one channel (specified by the channel attribute) is read
-        relative to the value of a second channel (implied by the chip's
-        design).
+        If ``True``, the device is operated in differential mode. In this mode
+        one channel (specified by the channel attribute) is read relative to
+        the value of a second channel (implied by the chip's design).
```

Comment:
I'm not sure about the removal of 'pseudo' here? AIUI the MCP30xx and MCP32xx use pseudo-differential mode (each channel's value must still be between GND and 3V3, both sharing the same GND), and it's only the MCP330xx chips that have proper differential mode (where it's only the voltage _difference_ between the two channels that is measured)?
But IANAEE - I Am Not An Electrical Engineer ;-)

Then again, to most users of GpioZero it probably doesn't make any difference. \*shrug\*